### PR TITLE
add admin for AddonRegionalRestrictions

### LINKED ISSUE #15968: Add admin for georestrictions
https://github.com/mozilla/addons-server/issues/15942 will initially land without an admin, but we should have one. It should be implemented as a regular django admin, with an email notification going out for every addition/modification/deletion that includes the details of the changes because it's so sensitive.

### PR DESCRIPTION
**Pull request by eviljeff**
fixes #15968 - minimally, it doesn't email yet.

### CODE REVIEW

**Review by bobsilverberg — APPROVED** [review]
Looks good, nice work @eviljeff. How is access to this admin function defined? Can anyone who can access the admin make these changes, or do they need to belong to a particular group?

r+wc

**Comment by eviljeff** [comment]
> Looks good, nice work @eviljeff. How is access to this admin function defined? Can anyone who can access the admin make these changes, or do they need to belong to a particular group?

The default for access to admin modules is only full admins (`*:*`) can access them.

File: src/olympia/addons/tests/test_admin.py [hunk 1]
```diff
@@ -726,3 +727,86 @@ def test_can_list_with_admin_curation_permission(self):
...
+        addon = addon_factory()
+        self.add_url = reverse('admin:addons_addonregionalrestrictions_add')
+
+        response = self.client.get(self.add_url, follow=True)
+        assert response.status_code == 200
+        assert pq(response.content)('#id_addon')  # addon input is editable
+
+        response = self.client.post(
+            self.add_url, {
+                'excluded_regions': '["de", "pt-BR"]',
```
Comment (willdurand) [inline src/olympia/addons/tests/test_admin.py:769]: Region code should be uppercase values only and I dont see that documented or enforced.

Comment (eviljeff) [reply src/olympia/addons/tests/test_admin.py:769]: I couldn't see that documented anywhere either - can you log an issue?

Comment (eviljeff) [reply src/olympia/addons/tests/test_admin.py:769]: I logged an issue https://github.com/mozilla/addons-server/issues/16023

**Pull request merged**